deploy.py

```diff
@@ -1,9 +1,11 @@
-import sys
+import os
+
+import argparse
 
 from bentoml.saved_bundle import load_bento_service_metadata
 
 from describe import describe_cloud_run
-from utils import run_shell_command, get_configuration_value, generate_cloud_run_names
+from utils import run_shell_command, get_configuration_value, generate_cloud_run_names,console
 
 
 def deploy_gcloud_run(bento_bundle_path, deployment_name, config_json):
```

Comment:
make sure you change the name too
deploy_gcloud_run -> deploy()
describe_gcloud_run -> describe()
update_gcloud_run -> update()
delete_gcloud_run -> delete()